components/tag/TagActionButton.vue

```diff
@@ -12,10 +12,16 @@ const emit = defineEmits<{
 const { client } = $(useMasto())
 
 const toggleFollowTag = async () => {
-  if (tag.following)
+  if (tag.following) {
```

Comment:
```suggestion
  tag.following = !tag.following
  if (tag.following)
```

Reply:
I put it below the if-else, as it should be only applied when the await doesn't throw